package.json

```diff
@@ -5,15 +5,15 @@
     "clean": "bsb -clean-world",
     "build": "bsb -make-world",
     "start": "bsb -make-world -w",
-    "test": "node src/Scratch.bs.js"
+    "test": "node src/scratch.bs.js"
   },
   "keywords": [
     "BuckleScript"
   ],
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "bs-platform": "^5.2.0",
+    "bs-platform": "^5.0.6",
```

Comment:
```suggestion
    "bs-platform": "^5.2.0",
```

Reply:
new bucklescript gets some really nifty compiler optimizations

Reply:
This will also reduce the diff footprint, because this string gets printed in the generated files.

Reply:
Done.